src/sidebar/components/thread-list.js

```diff
@@ -13,14 +13,25 @@ import {
 
 import ThreadCard from './thread-card';
 
+/** @typedef {import('../util/build-thread').Thread} Thread */
+
 // The precision of the `scrollPosition` value in pixels; values will be rounded
 // down to the nearest multiple of this scale value
 const SCROLL_PRECISION = 50;
 
 function getScrollContainer() {
-  return document.querySelector('.js-thread-list-scroll-root');
+  const container = document.querySelector('.js-thread-list-scroll-root');
+  if (!container) {
+    throw new Error('Scroll container is missing');
```

Comment:
I'm not really sure if this can even happen, but its simple enough to throw an error. I suppose we could cast too, but this felt a bit safer.

Reply:
👍 - It makes sense to check here, since the scroll container is not created by the component itself. Ideally it would be better if the scroll container were part of the component, but when this was set up there were good reasons not to do that.